### ISSUE TITLE
Navigation quirks

### ISSUE THREAD

**Issue by cloudhead** [radicle-dev/radicle-interface#747]
I noticed two quirks in project navigation:

1. When navigating away from a patch page, the `?tab=files` query sticks around
2. It's possible to view patches/issues etc. with a remote selected. This doesn't make sense, as patches are loaded with all remotes always. So selecting a remote should go back to source browsing always. Having a remote selected should only work in the source code or commits tab, basically anything code related.

**Comment by cloudhead** [comment]
Yes, I think that would make sense. I'm just not sure how it's going to look without some good design work :sweat_smile:

**Comment by rudolfs** [comment]
I think it could work 🤷
You can play around with it here if you like: https://github.com/radicle-dev/radicle-interface/tree/rudolfs/header-hierarchy-mockup

<img width="1840" alt="Screenshot 2023-05-12 at 10 00 34" src="https://github.com/radicle-dev/radicle-interface/assets/158411/14deee0c-70d1-4478-8394-cda69132e70e">
<img width="1840" alt="Screenshot 2023-05-12 at 10 00 39" src="https://github.com/radicle-dev/radicle-interface/assets/158411/a4328f26-a2a9-4cfe-8364-3f112cefcb03">
<img width="1840" alt="Screenshot 2023-05-12 at 10 01 06" src="https://github.com/radicle-dev/radicle-interface/assets/158411/efedb5ab-c96a-4167-a9d6-247dc02b0a12">

**Comment by cloudhead** [comment]
I would start like this, but still needs work eventually:
<img width="1480" alt="image" src="https://github.com/radicle-dev/radicle-interface/assets/40774/b3737b26-277a-4a18-935c-676e3ff8235d">

* Removed content bg
* Removed content padding
* Restored button size
* Added 2rem bottom margin to header
